packages/rancher-istio/rancher-istio/charts/values.yaml

```diff
@@ -66,6 +66,9 @@ global:
     enabled: true
   rbac:
     pspEnabled: true
+  # Set it false before installing/upgrading istio in an air-gapped setup. It's set to true by default.
+  configValidation:
+    enabled: true
```

Comment:
I don't think this change makes sense. Where is this value being used by the Helm chart? Has this change been tested?

I'd suspect you need to add this value into the global config of the IstioOperator resource, which is found in https://github.com/rancher/charts/blob/eaba8696d0e569a64c45ef7f610b501528daae9d/charts/rancher-istio/100.1.2%2Bup1.11.7/configs/istio-base.yaml#L102-L110.

But there's no field there actually referencing this value that was set. Also, I'm not sure if this field should belong in `.values.global` of the Helm chart itself.

Reply:
Hey, yeah I think this should be set in [charts/charts/rancher-istio/100.1.2+up1.11.7/configs/istio-base.yaml](https://github.com/rancher/charts/blob/eaba8696d0e569a64c45ef7f610b501528daae9d/charts/rancher-istio/100.1.2%2Bup1.11.7/configs/istio-base.yaml#L102-L110) as well which will use the value from values.yaml while installing istio. 
We can't actually add any new repo in apps and marketplace on an air-gapped setup, so I tested it this way: while installing Istio, in the yaml, I added the below config (set it as false because, it's set to true by default.) inside global and tried to install istio. Then again uninstalled and reinstalled/upgraded to test if it's working. Istio installation was getting installed/upgraded successfully. 
```
configValidation:
    enabled: false
```
Without setting the `configValidation` to false, only on airgapped setup it's breaking. If we set it to false while installing or upgrading it's getting installed successfully.

Reply:
I see changes were added to the istio-base.yaml, resolving this comment